website/docs/guides/full-text-search/1-drop-books-collection.request.js

```diff
@@ -0,0 +1,3 @@
+db.runCommand({
+  drop: 'books'
+})
```

Comment:
Why is `1-drop-books-collection` not referenced in `full-text-search.mdx` but generating it here?

Reply:
Not really. The essence of the drop command is to insert a new books fixture into it without affecting the preset books fixture. Running the commands without dropping them will lead to duplicate`_id`.

We decided that some operations won't need all the fields, so we can just insert new ones that can be tested with CTS. https://github.com/FerretDB/FerretDB/pull/5131#discussion_r2077991189

Reply:
ok

Reply:
There is a problem, though. The CTS tool inserts data from fixtures first, then runs test suites in non-deterministic order. Some drop collections and insert their own data, while others rely on data in the fixture. That makes the CTS tool run on CI fail sometimes.

The easiest solution seems to be to drop books.fixture.json completely. I believe we even discussed that. So why wasn't it done?

@Fashander @noisersup please discuss that on the daily doc stnc

Reply:
I believe that shouldn't be a problem, as we setup fixture before running each test suite (we drop and insert collection before each json file).

Reply:
@noisersup So if I understand correctly, `full-text-search` contains a test suite with `1-drop-books-collection`, `2-insert-data`, `3-create-text-index`, ...,  `7-relevance-score`.

Does it mean the fixture `fixtures/books.js` is inserted for this test suite, then the test suite `full-text-search` starts which runs `1-drop-books-collection`, `2-insert-data`, `3-create-text-index`, ...,  `7-relevance-score`?

If so, `1-drop-books-collection` and `2-insert-data` are redundant and not needed right? Bare in mind, I might be misunderstanding how it works

Reply:
So I understand this now, `fixtures/books.js` contain more fields than needed, for example vector fields.
https://github.com/FerretDB/FerretDB/blob/14feeaaf717b45ca5cc3633f53a09c781eeda04c/website/docs/guides/fixtures/books.js#L78-L83

For full-text-search, it wants fewer fields, so generated fixtures were deleted and new simpler fixtures were inserted. I tend to think it might be easier to add another fixture with smaller number of fields, but current way serves the need.